apps/ocean-api/__tests__/modules/PlaygroundModule.test.ts

```diff
@@ -0,0 +1,48 @@
+import { OceanApiTesting } from '../../testing/OceanApiTesting'
+import waitForExpect from 'wait-for-expect'
+
+describe('regtest', () => {
+  const apiTesting = OceanApiTesting.create()
+
+  beforeEach(async () => {
+    apiTesting.playgroundEnable(false)
+    await apiTesting.start()
+  })
+
+  afterEach(async () => {
+    await apiTesting.stop()
+  })
+
+  it('block count should not increment', async () => {
```

Comment:
`nit:` do we have any conventions for `it` method name argument in our tests? In this case we could consider
```
it('should not increment block count')
```
being potentially more correct sentence structure here.

Reply:
```suggestion
  it('should not increment block count', async () => {
```

Thanks! You can also add suggestions for me to batch commit in